tests/gridworld_test.py

```diff
@@ -150,8 +154,8 @@ def test_equality(self):
     def test_dumploadloop(self):
         assert self.X == gw.GridWorld(self.X.dumps())
 
-    @unittest.skipIf(psutil.virtual_memory().total < 64e9,
```

Comment:
Following this change, `import psutil` can be removed from the file.

Reply:
I could rebase to remove the `import psutil`.

Reply:
I think it is enough to add another commit, no rebase, unless you prefer rebase.

Reply:
Then, we can also remove `psutil` from requirements.txt.

Reply:
It was introduced in commit 0c1b77ce5355f272a2ddece2f8bccdb42f479f35.

To be clear, I am not opposed to using `psutil`. We might have reason to depend on it again in the future. This change is simply to keep the list of dependencies up-to-date.